src/pages/Clients/index.js

```diff
@@ -0,0 +1,229 @@
+import React, { useState, useEffect } from "react";
+import axios from "../../config/API";
+import { animateScroll as scroll } from "react-scroll";
+import { getAuthorizationHeader } from "../../services/Authentication";
+import { Formik } from "formik";
+import FormMultipleSelect from "../../components/FormComponents/FormMultipleSelect";
+import {
+  realEstateTypes,
+  neighborhoods,
+} from "../../utils/FormHelpers/form-data";
+import { formatCollectionForReactSelect } from "../../utils/FormHelpers/FormatCollectionForReactSelect";
+import { FormTextField } from "../../components/FormComponents/FormTextField";
+import parseJwt from "../../utils/parseJwt";
+import CreateNewClient from "../../utils/CreateNewClient";
+
+function AddClient({
+  setNewClientData,
+  newClientData,
+  setIsFormShowing,
+  clientsData,
+  setClientsData,
+}) {
+  const handleSubmit = async (values) => {
+    const token = window.localStorage.getItem("userToken");
+    const { userId } = parseJwt(token);
+    setNewClientData(mapValues(values, userId));
```

Comment:
Here you can just use a new constant, because this is information is not used outside of the function
```suggestion
    const mappedData = mapValues(values, userId));
```